pkg/cmd/roachtest/test_test.go

```diff
@@ -482,3 +483,50 @@ func TestNewCluster(t *testing.T) {
 		})
 	}
 }
+
+func TestGCESameDefaultZone(t *testing.T) {
```

Comment:
Let's add a comment to make it clear that it's a regression test, along with the GH issue reference.